.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -96,18 +96,32 @@ jobs:
 
           return doesExist ? 1 : 0;
 
+    # put the markdown into a variable
+    - name: Levitate Markdown
+      id: levitate-markdown
+      if: steps.levitate-run.outputs.shouldSkip != 'true'
+      run: |
+          if [ -f "${ARTIFACT_FOLDER}/levitate.md" ]; then
```

Comment:
Not sure I'm reading all this correctly but...

`ARTIFACT_FOLDER: '${{ github.workspace }}/tmp'` means this will be checking for `whatever_workspace_is/tmp/levitate.md` but it's echo'd like `echo $GITHUB_LEVITATE_MARKDOWN >./levitate/levitate.md`? Are these supposed to be the same location? 🤔

Reply:
>echo $GITHUB_LEVITATE_MARKDOWN >./levitate/levitate.md

So this part is a bit complicated...

* The build workflow creates a "levitate" (contains result.json and levitate.md) artefact that is uploaded to github artefacts. 
* The report workflow downloads it and uncompress it in `${{ github.workspace }}/tmp` which is how it becomes available there.